requirements.txt

```diff
@@ -1,4 +1,4 @@
--e git+https://github.com/oda-hub/dispatcher-app.git@master#egg=cdci_data_analysis[test]
+-e git+https://github.com/oda-hub/dispatcher-app.git@fix-numpy-alises#egg=cdci_data_analysis[test]
```

Comment:
ok but we need to revert it after https://github.com/oda-hub/dispatcher-plugin-integral-all-sky/issues/20